paparazzi/src/test/java/app/cash/paparazzi/HtmlReportWriterTest.kt

```diff
@@ -62,13 +62,13 @@ class HtmlReportWriterTest {
         """
         |window.runs["run_one"] = [
         |  {
-        |    "file": "images/$anyImageHash.png",
         |    "name": "loading",
+        |    "testName": "app.cash.paparazzi.CelebrityTest#testSettings",
+        |    "timestamp": "2019-03-20T10:27:43.000Z",
         |    "tags": [
         |      "redesign"
         |    ],
-        |    "testName": "app.cash.paparazzi.CelebrityTest#testSettings",
-        |    "timestamp": "2019-03-20T10:27:43.000Z"
+        |    "file": "images/$anyImageHash.png"
```

Comment:
discussed w @hzsweers.  apparently, this version of Moshi now preserves property appearance ordering

Reply:
Ah, so 1.8.0 explicitly sorted them?

Reply:
It appears so.  I didn't dig in too much to see where the behavior change occurred.